kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/ManageSyncSchedule.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+  >
+    <KPageContainer>
+      <KGrid gutter="48">
+        <KGridItem>
+          <h1>{{ $tr('manageSyncTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ $tr('introduction') }}</p>
+        </KGridItem>
+
+        <KGridItem :layout8="{ span: 4 }" :layout12="{ span: 6 }">
+          <p>{{ $tr('homeFacility') }}</p>
+        </KGridItem>
+        <KGridItem
+          :layout="{ alignment: 'right' }"
+          :layout8="{ span: 4 }"
+          :layout12="{ span: 6 }"
+        >
+          <KButton
+            @click="deviceModal = true"
+          >
+            {{ $tr('addDevice') }}
+          </KButton>
+        </KGridItem>
+      </KGrid>
+
+      <CoreTable>
+        <template #tbody>
+          <tbody class="table">
+            <tr>
+              <th>{{ $tr('deviceName') }}</th>
+              <th>{{ $tr('Schedule') }}</th>
+              <th>{{ $tr('Status') }}</th>
+              <th></th>
```

Comment:
I think it would be good to still have a header here, even if it is not visible. This will help screen reader users, who hear what is on the page, rather than read it. We can ask @radinamatic what a suitable option might be - perhaps the `visually-hidden` class and a description of what the contents of the column is? Here is [some general information about tables and accessibility](https://webaim.org/techniques/tables/) (it is not specific documentation to this case, or a suggestion of a solution - just for your learning!)